src/store/reducers/genres.js

```diff
@@ -0,0 +1,40 @@
+import {
+  GET_GENRES_BY_ID_PENDING,
+  GET_GENRES_PENDING,
+  GET_GENRES_SUCCESS,
+  GET_GENRES_ERROR,
+} from '../actionTypes';
+import { normalizeData } from '../../utils';
+
+const defaultState = {
+  byId: {},
+  isLoading: false,
+};
+
+export const genres = (state = defaultState, { type, payload }) => {
+  switch (type) {
+    case GET_GENRES_BY_ID_PENDING:
+      return { ...state, isLoading: true };
+    case GET_GENRES_PENDING:
```

Comment:
you can combine this two actions `GET_GENRES_BY_ID_PENDING` and `GET_GENRES_PENDING`.
```js
case GET_GENRES_BY_ID_PENDING:
case GET_GENRES_PENDING:
  return { ...state, isLoading: true };
```